guardian/newsletters-nx
# Replaced Wizard buttons with styled MUI buttons...

## What does this change?

Updates the Wizard buttons and replaces to `Red`, `Green` types with some action themed type:

```javascript
export type WizardButtonType =
	| 'NEXT'
	| 'PREVIOUS'
	| 'EDIT'
	| 'SKIP'
	| 'LAUNCH'
	| 'CANCEL';
```

This PR also removes refs to `Demo` from buttons and routes with the exception of `Demo form` which really does look like a demo. 

## How to test

Check the code out and verify that the existing functionality is as before. The behaviour should not have changed - only some of the routes.

## Images

<img width="1305" alt="Screenshot 2023-05-01 at 21 26 56" src="https://user-images.githubusercontent.com/3277259/235525794-f94f4013-478a-4535-95f4-66437e8967c7.png">

<img width="1314" alt="Screenshot 2023-05-01 at 21 32 14" src="https://user-images.githubusercontent.com/3277259/235526014-3427b5f9-8cf3-4bcb-b2a2-f53000fd929a.png">
